### PR TITLE
Adding build targets to create adoc bundle and adoc variables file for Minishift version

### PR DESCRIPTION
**Pull request by hferentschik**
Addresses issue #560, #536

### CODE REVIEW

**Comment by budhrg** [comment]
Hi @hferentschik , I am able to build the tar using `make gen_adoc_tar`. However, not sure about the tar content.

```
$ ls
minishift-adoc.tar  

$ sudo tar -xf minishift-adoc.tar  (need to do sudo)

$ ls
minishift-adoc.tar  source

$ tree .
.
├── minishift-adoc.tar
└── source
    ├── command-ref
    │   ├── minishift.adoc
    │   ├── minishift_config.adoc
    │   ├── minishift_config_get.adoc
    │   ├── minishift_config_set.adoc
    │   ├── minishift_config_unset.adoc
    │   ├── minishift_config_view.adoc
    │   ├── minishift_console.adoc
    │   ├── minishift_delete.adoc
    │   ├── minishift_docker-env.adoc
    │   ├── minishift_ip.adoc
    │   ├── minishift_logs.adoc
    │   ├── minishift_openshift.adoc
    │   ├── minishift_openshift_config.adoc
    │   ├── minishift_openshift_config_set.adoc
    │   ├── minishift_openshift_config_view.adoc
    │   ├── minishift_openshift_list-versions.adoc
    │   ├── minishift_openshift_restart.adoc
    │   ├── minishift_openshift_service.adoc
    │   ├── minishift_openshift_service_list.adoc
    │   ├── minishift_ssh.adoc
    │   ├── minishift_start.adoc
    │   ├── minishift_status.adoc
    │   ├── minishift_stop.adoc
    │   └── minishift_version.adoc
    ├── development
    │   └── developing.adoc
    ├── minishift.adoc
    ├── _topic_map.yml
    └── using
        ├── docker-machine-drivers.adoc
        ├── interacting-with-openshift.adoc
        ├── managing-minishift.adoc
        └── troubleshooting.adoc

4 directories, 32 files
```

File: Makefile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = 1.0.0-beta.5
+MINISHIFT_VERSION = 1.0.0-beta.5
```
Comment (budhrg) [inline Makefile]: :+1:  Good to be specific

Comment (hferentschik) [inline Makefile]: yeah, we should have done this earlier. Basically at them moment we started to have more than one version. Really, we would also in the Go code rename version to minishiftVersion, but I left that for another time.

**Comment by hferentschik** [comment]
> However, not sure about the tar content.

You think something is wrong? Content looks good to me. Any concerns?

**Comment by hferentschik** [comment]
>> However, not sure about the tar content.

> You think something is wrong? Content looks good to me. Any concerns?

That said, I expect that we might have to adjust the content a couple of times. For example, the content of __topic_map.yml_ might not be quite right yet. Also other things might need adjustment. The main point is to create an initial tarball and put it on a publicly accessible spot. Then we can start the integration process with docs.openshift.org.

**Comment by hferentschik** [comment]
@budhrg rebased.

**Comment by hferentschik** [comment]
@budhrg pushed another update which is tested against  https://github.com/openshift/openshift-docs.git

If you like, you can confirm by building the tarball and then:

    $ git clone https://github.com/openshift/openshift-docs.git
    $ cd openshift-docs
    $ gem install bundler
    $ bundle install
    $ mkdir minishift
    $ cd minishift
    $ cp <location of tarball> .
    $ tar -xvf minishift-adoc.tar --strip 1
    $ cat _topic_map.yml >> ../_topic_map.yml
    $ cd ..
    $ rake build
    $ open _preview/openshift-origin/latest/welcome/index.html

**Comment by budhrg** [comment]
@hferentschik All look good :+1: except the overview page as it couldn't load the variables and hence version. See below.

minishift_doc

**Comment by hferentschik** [comment]
what's the content of your minishift directory within the Origin docs checkout? And what does minishift/variables.adoc contain?

**Comment by hferentschik** [comment]
I did a rebuild from scratch and get the following in the Minishift overview file:

"Welcome to Minishift version 1.0.0-beta.5!"

**Comment by budhrg** [comment]
@hferentschik , Now working fine ! :+1: Might be copied old tar file :)

**Comment by hferentschik** [comment]
> Now working fine ! Might be copied old tar file :)

Awesome, let's merge ;-)

**Comment by budhrg** [comment]
Merged and Thanks!

**Pull request closed without merging**